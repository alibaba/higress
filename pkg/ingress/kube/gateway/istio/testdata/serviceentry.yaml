apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: gateway
  namespace: higress-system
spec:
  gatewayClassName: higress
  listeners:
  - name: default
    port: 80
    protocol: HTTP
    allowedRoutes:
      namespaces:
        from: All
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: HTTPRoute
metadata:
  name: http
  namespace: default
spec:
  parentRefs:
  - name: gateway
    namespace: higress-system
  rules:
  -
    backendRefs:
    - kind: Hostname
      group: networking.istio.io
      name: google.com
      port: 80
---
