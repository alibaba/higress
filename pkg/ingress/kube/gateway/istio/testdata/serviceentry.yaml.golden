apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/parents: Gateway/gateway/default.higress-system
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway-default
  namespace: higress-system
spec:
  selector:
    higress: higress-system-higress-gateway
  servers:
  - hosts:
    - '*/*'
    port:
      name: default
      number: 80
      protocol: HTTP
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parents: HTTPRoute/http.default
    internal.istio.io/route-semantics: gateway
  creationTimestamp: null
  name: http-0-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  gateways:
  - higress-system/gateway-istio-autogenerated-k8s-gateway-default
  hosts:
  - '*'
  http:
  - name: default/http
    route:
    - destination:
        host: google.com
        port:
          number: 80
---
